docs/dev.md

```diff
@@ -1,4 +1,6 @@
 # Run Drava in Development Mode
+?> This page illustrates how to run Drava in your local computer in the development mode.
+
 First, download or clone our github repo at https://github.com/wangqianwen0418/DRAVA.
```

Comment:
Should we remove this link before submission?

Reply:
Good point. I have created another branch `anonymous` and will host the demo using that branch at https://anonymous.dcfg7f17k3km1.amplifyapp.com/ for the review process.

Reply:
@wangqianwen0418 I think it is well-anonymized. Your name is shown in `package.json` and Google Docs, but I think that should be fine.

![Screen Shot 2022-09-02 at 12 03 13](https://user-images.githubusercontent.com/9922882/188194566-929291e3-0d2a-4a3a-b149-0af032a710c1.png)

![Screen Shot 2022-09-02 at 12 01 41](https://user-images.githubusercontent.com/9922882/188194568-1fc66cdb-438b-48b0-bd42-35f4cabb2f71.png)